iceberg/src/test/scala/org/apache/spark/sql/delta/ConvertToIcebergSuite.scala

```diff
@@ -109,6 +111,7 @@ class ConvertToIcebergSuite extends QueryTest with Eventually {
       runDeltaSql(
         s"""CREATE TABLE `${testTableName}` (col1 INT) USING DELTA
            |TBLPROPERTIES (
+           |  'delta.enableIcebergCompatV2' = 'true',
```

Comment:
@Fokko already fixed this in https://github.com/delta-io/delta/pull/3330/files, I think we'd want to get that in first. These existing tests do not pass at the moment without this.

Reply:
I don't think those tests run in CI because it will skip over them when the Docker container is not there. Nevertheless, I think it would be good to get #3330 in to simplify testing.